modules/test/playwright/pages/document-library-web/DocumentLibraryPage.ts

```diff
@@ -27,14 +28,59 @@ export class DocumentLibraryPage {
 			`/group${siteUrl || '/guest'}${PORTLET_URLS.documentLibrary}`
 		);
 	}
+	async assertPrivateContentIcon() {
+		await expect(
+			this.page
+				.getByLabel('Not Visible to Guest Users')
+				.locator('use')
+				.last()
+		).toBeVisible({timeout: 1000});
+	}
+
+	async changeTab(tabName: string) {
+		await this.page.getByRole('link', {name: tabName}).click();
+	}
+
+	async changeView(viewName: string) {
+		await this.page
+			.getByLabel('Select View, Currently Selected: ')
+			.waitFor();
+		await this.page.getByLabel('Select View, Currently Selected: ').click();
+		await this.page.getByRole('menuitem', {name: viewName}).click();
```

Comment:
Use the `clickAndExpectToBeVisible` around here we are having the same problem, the `this.page.getByRole('menuitem', {name: viewName})` isn't visible when we launch the click